### ISSUE TITLE
Update Cypress

### ISSUE THREAD

**Issue by marlass** [SAP/cloud-commerce-spartacus-storefront#3095]
Cypress released 3.3.1 some time ago.
Might be worth upgrading to potentially fix some bugs and improve perf.

**Comment by marlass** [comment]
After #5780 merge we can update to 3.8.0 version

**Comment by marlass** [comment]
Duplicate of #5922 
Closing
